### ISSUE TITLE
GraphQL Pro Dashboard: ActiveRecord errors on first use

### ISSUE THREAD

**Issue by sj26** [rmosolgo/graphql-ruby#1138]
I added the fresh GraphQL Pro Dashboard to our routes this morning and was greeted with an ActiveRecord error about missing a `graphql_clients` table:

```
PG::UndefinedTable at .../graphql/dashboard/clients
ERROR:  relation "graphql_clients" does not exist
LINE 8:                WHERE a.attrelid = '"graphql_clients"'::regcl...
                                          ^
```

I believe this is because `GraphQL::Pro::Dashboard::ApplicationComponent#home` redirects to the first "enabled" component, which is `GraphQL::Pro::Dashboard::OperationStoreComponent` because it is implicitly enabled when `GraphQL::Pro::OperationStore` is defined, which is always true if `ActiveRecord` is defined when `graphql/pro` is required (which it is, in our application).

I was able to fix this by patching `GraphQL::Pro::Dashboard::OperationStoreComponent#enabled?` to include a schema metadata check:

```ruby
        def enabled?
          defined?(GraphQL::Pro::OperationStore) && @schema.metadata.include?(GraphQL::Pro::OperationStore::KEY)
        end
```

Now I get a nice "No Active Components" page which gives me somewhere to get started on our journey to subscriptions. ^_^

Hotfix:

```ruby
GraphQL::Pro::Dashboard::OperationStoreComponent.class_eval do
  def enabled?
    defined?(GraphQL::Pro::OperationStore) && @schema.metadata.include?(GraphQL::Pro::OperationStore::KEY)
  end
end
```

**Comment by rmosolgo** [comment]
Thanks for sharing the fix! I'll fix up the tests so they properly cover this, and release a patch soon.

**Comment by daqo** [comment]
We also faced the issue. The same hotfix resolved the situation for us.

**Comment by rmosolgo** [comment]
Thanks again for the details, fixed in 1.6.2!
